src/validation_workflow/yum.py

```diff
@@ -0,0 +1,26 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from .download_utils import DownloadUtils
+from .validation import Validation
+
+
+class Yum(Validation, DownloadUtils):
+    @classmethod
+    def get_architectures(cls) -> list:
+        pass
+
+    @classmethod
+    def download_urls(cls, projects: list, version: str, platform: str) -> None:
+        for project in projects:
+            url = f"https://artifacts.opensearch.org/releases/bundle/{project}/2.x/{project}-2.x.repo"
```

Comment:
This is won't be true everytime. For example, for 1.x version this static link will point to `1.x`. Calculate it using the user entered `version` value. For example, for 3.0.0 version the url will be `3.x`
cc: @peterzhuamazon